### ISSUE TITLE
Rule no-process-exit in the recommended rules

### ISSUE THREAD

**Issue by doberkofler** [mysticatea/eslint-plugin-node#228]
According to the documentation  there is a rule `node/no-process-exit` but in the recommended rules it is named `no-process-exit`.

**Comment by mysticatea** [comment]
Yes. Because I had avoided breaking changes at that time. I will merge https://github.com/mysticatea/eslint-plugin-node/pull/211 after ESLint gets supporting Optional Chaining syntax, then I will release a new major version with the new recommended config and new checks for the new syntax.

**Comment by thernstig** [comment]
@EnzoMartin any chance you would want to create a PR for your workaround?

**Comment by EnzoMartin** [comment]
> @EnzoMartin any chance you would want to create a PR for your workaround?

That's just my own personal workaround until the rules were updated, I referenced the issue in case it gets solved so I can update my personal package 😊 Appreciate the thought though!
